include/evmc/evmc.h

```diff
@@ -991,7 +991,12 @@ struct evmc_instance
      * Optional pointer to function setting the EVM instruction tracer.
      *
      * If the EVM does not support this feature the pointer can be NULL.
+     *
+     * @deprecated
+     * The tracing API from EVMC 6 has been deprecated as there have been some
+     * design flaws discovered. New API is expected to be introduced in future.
      */
+    EVMC_DEPRECATED
```

Comment:
We cannot put [[deprecated]] attribute here because it makes the `evmc_instance` non-constuctable in some compilers: https://circleci.com/gh/ethereum/evmc/13261.